src/include/list_link.h

```diff
@@ -78,12 +78,12 @@ typedef pbs_list_link pbs_list_head;
 
 /* function prototypes */
 
-extern void insert_link(pbs_list_link *old, pbs_list_link *newp, void *pobj, int pos);
-extern void append_link(pbs_list_head *head, pbs_list_link *newp, void *pnewobj);
```

Comment:
Sorry for making you do this change back, but from your build report i figured out the problem. Some other tools that use this header but uses C++ to compile are failing. So let's go ahead and change it back to what you had before, which is newp.

Reply:
Done.